zerver/lib/push_notifications.py

```diff
@@ -629,7 +629,7 @@ def process(elem: lxml.html.HtmlElement) -> str:
         return plain_text
 
     if settings.PUSH_NOTIFICATION_REDACT_CONTENT:
-        return "***REDACTED***"
+        return _("***REDACTED***")
```

Comment:
Is this running in a context where `_` knows which language to use?

Reply:
I have no idea! Do you know how I might start to find out? (context: this is effectively day one for me contributing to zulip)

Reply:
Hmm. `get_mobile_push_content` is called from `get_message_payload_apns` and `get_message_payload_gcm`, which are called from `handle_push_notification`, which is called from `PushNotificationsWorker`, which is registered in `worker_classes` by `assign_queue`, and thereby run from a `process_queue` management command. I don’t see anything in this path that would activate a translation (`django.utils.translation.activate`).